.bsp/sbt.json

```diff
@@ -1 +1 @@
-{"name":"sbt","version":"1.10.1","bspVersion":"2.1.0-M1","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk-11.jdk/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/bilal.amiry/Library/Application Support/JetBrains/IdeaIC2024.1/plugins/Scala/launcher/sbt-launch.jar","-Dsbt.script=/opt/homebrew/bin/sbt","xsbt.boot.Boot","-bsp"]}
```

Comment:
Look into .gitignore files these can be added to you git repos so you do not push all of these files. For example: https://alvinalexander.com/source-code/scala/sample-gitignore-file-scala-sbt-intellij-eclipse/?utm_content=cmp-true

Reply:
I have updated gitignore file.